libs/utils/platforms/hikey960.json

```diff
@@ -0,0 +1,37 @@
+{
+	// HiKey960 boards have two clusters
+	// 4 Cortex A73 + 4 Cortex A53 big.LITTLE architecture
+	"board" : {
+		"cores" : [
+			"A53_0", "A53_0", "A53_0", "A53_0",
+			"A73_1", "A73_1", "A73_1", "A73_1"
+		],
+		"big_core" : "A73_1",
+		"modules"  : [ "bl", "cpufreq", "cpuidle" ]
+	},
+
+    "nrg_model" : {
+        "little" : {
+            "cluster" : {
+                "nrg_max" :   70,
+            },
+	        "cpu" : {
+                "nrg_max" :   1210,
```

Comment:
The hi3660-sched-energy.dtsi file is for the HiKey960 - the CA72 instead of CA73 mistype is a common error with that board (same thing in the Trusted Firmware, it has been fixed actually).